dlrover/python/master/node/job_context.py

```diff
@@ -45,6 +45,15 @@ def __init__(self):
         self._pre_check_status: str = PreCheckStatus.CHECKING
         self._request_stopped = False
         self._locker = threading.Lock()
+        self._job_stage = JobStage.JOB_INIT
```

Comment:
Should merge these 2 fields into 1:
![image](https://github.com/user-attachments/assets/e8d2405d-3bc9-4fb6-ae48-6ef7b1f88048)